.github/workflows/env_test.yml

```diff
@@ -28,4 +28,4 @@ jobs:
         env:
           NAS_CREDS: ${{ secrets.NAS_CREDS }}
         run: |
-          poetry run python -m pytest pyha_tests
\ No newline at end of file
+          poetry run python -m pytest pyha_tests -rP
```

Comment:
What is this flag for?

Reply:
collects and prints stdout from test after the test ends. IE log/print statements

Reply:
Sounds good, probably good to put for other repos.  Can you add to python-repo-example?